### ISSUE TITLE
Double clicking on empty cells in metadata table throws console errors

### ISSUE THREAD

**Issue by jmurugan-fzj** [PASTA-ELN/pasta-eln#305]
**Describe the bug**
Double clicking any randomly selected cells in the metadata table of data hierarchy editor throws console errors.
Reproducible: Occasional

**Steps To Reproduce**
1. Run the latest version of PASTA application.
2. Go to System | Data hierarchy editor (or press F8)
3. Switch items between "Data Type" combo-box (eg. Structure level 0, measurement, sample etc.)
4. ![image](https://github.com/PASTA-ELN/pasta-eln/assets/141253011/c0710081-e064-4611-be9e-8e451eac8763)
5. Double click on any cells in the metadata table randomly (empty or non-empty)
6. ![image](https://github.com/PASTA-ELN/pasta-eln/assets/141253011/d749c3f2-9609-4496-b9d7-df8c9e113463)
7. See the following type of error been thrown in the console
8. ![image](https://github.com/PASTA-ELN/pasta-eln/assets/141253011/6ac3f61e-4071-4226-8868-1af9e7e5e0f5)

**Expected behavior**
Double click on any empty or normal cell should not throw any console errors.

**Screenshots**
Given above.

**Desktop (please complete the following information):**
 - OS: Ubuntu 24.04 LTS
 - Browser Chrome
 - PASTA ELN Version: Latest version adapted for python3.12.4
 - Python Version: Python 3.12.4

**Additional context**
Similar issue has been reported already related to specific pyqt versions: https://stackoverflow.com/questions/46383409/qaccessibletablechild-invalid-index-in-empty-directory-with-qtableview-and-qf
https://forum.qt.io/topic/23872/5-0-1-cannot-create-accessible-interface-for-object-warnings/2

**Comment by HTsybenko** [comment]
This is indeed an occasional bug, and occurs rather rarely for me as well (2.6.0b1).

**Comment by HTsybenko** [comment]
@jmurugan-fzj Yes, I agree. I would also prioritize the related issue #306, since it does not allow metadata entries in the affected fields.

**Comment by jmurugan-fzj** [comment]
@HTsybenko The fix for the issue #306 is currently in review now..

**Comment by SteffenBrinckmann** [comment]
Reimplemented new version, not an issue there.
